rules/manager.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
-	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
unrelated?

Reply:
not related. it is removed by `go mod tidy`